build.zig

```diff
@@ -332,6 +332,9 @@ pub fn MicroBuild(port_select: PortSelect) type {
             /// If set, overrides the `linker_script` property of the target.
             linker_script: ?LazyPath = null,
 
+            /// If set, overrides the default `entry` property of the arget.
+            entry: ?Build.Step.Compile.Entry = null,
```

Comment:
This is in the firmware options... maybe this doesn't make sense to be here, but we probably at least want it in the target.